src/main/java/com/github/skjolberg/packing/Container.java

```diff
@@ -70,10 +70,6 @@ public Placement get(int level, int placement) {
 		return levels.get(level).get(placement);
 	}
 
-	public void validateCurrentLevel() {
```

Comment:
This is in use during testing sometimes.

Reply:
restored above